department-of-veterans-affairs/va.gov-cms
# VAGOV-3951

**About:**
- Adding patch to fix mod sidebar, and removing unused patches.


**Screenshot of expected behavior:**
![Screenshot_2019-06-13_13-14-02](https://user-images.githubusercontent.com/2404547/59453109-2cc66880-8ddd-11e9-812d-a1c0daf46088.png)

**QA:**
- As an admin, visit the node view of a page that has a draft / is in draft status, e.g.: `check-your-va-claim-or-appeal-status`
- Click the `DRAFT` button in the admin menu.
- Visually verify that the moderation sidebar appears
